tests/setup.js

```diff
@@ -3,6 +3,9 @@ import config from 'config';
 import areIntlLocalesSupported from 'intl-locales-supported';
 import values from 'object.values';
 
+import 'jest-enzyme';
```

Comment:
Aha, just checked add stuff like `.toIncludeText('foo')` needs this side-effect import to monkey-patch all the `expect()` objects.

I've added some docs around it to explain why we do it.